src/System.Runtime.Extensions/tests/System/EnvironmentTests.cs

```diff
@@ -390,20 +390,7 @@ public static IEnumerable<object[]> EnvironmentVariableTargets
                     return false;  // On Unix, registry-based environment api's won't throw a SecurityException - they just eat all writes.
                 if (!PlatformDetection.IsWinRT)
                     return false;  // On non-appcontainer apps, these won't throw (except writes to Target.Machine on non-elevated but that's accounted for separately.)
-
-                try
-                {
-                    Environment.GetEnvironmentVariables(EnvironmentVariableTarget.User);
-                }
-                catch (SecurityException)
-                {
-                    return true; // AppX registry exemptions not yet granted (at least on this build.)
-                }
-                catch
-                {
-                    return false; // Hmm... some other exception. We'll enable the individual tests and let them report it...
-                }
-                return false;
+                return true;  // Inside app-containers, no plans to support registry-based environment lookups.
```

Comment:
>Perhaps we should consider what the behavior inside an appcontainer should be, though? 
>Maybe instead of throwing a SecurityException, it should just present a black hole like Unix does?

@danmosemsft - what do you think? It's what we used to do for Process environment variables pre-NS2.0 and it seems preferable to having a second set of degraded behaviors...